spec/integration/rack_spec.rb

```diff
@@ -49,6 +49,8 @@
 
   let(:s3_double) { instance_double("Alephant::Storage", :get => content) }
 
+  let(:not_modified_status_code) { Alephant::Broker::Response::Base::NOT_MODIFIED_STATUS_CODE }
```

Comment:
@dazoakley, I couldn't use `described_class::NOT_MODIFIED_STATUS_CODE` here because the described class isn't the response. It's `Alephant::Broker::Application`.